docs/src/inputs/input/examples/exampleInputOnFocus.jsx

```diff
@@ -0,0 +1,14 @@
+import { Input } from 'react-cm-ui';
+import React from 'react';
+
+function ExampleInputOnFocus() {
+    const onInputFocus = () => {
+        console.log('Your focus determines your reality.');
```

Comment:
We should probably suppress ESLINT `no-console` ;-)

Reply:
Please check this commit https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/8429b8ffd987325a6ae504db3aa2341b60de60ef

Reply:
Looks good.  Resolving.